src/component/tooltip/TooltipView.ts

```diff
@@ -509,9 +509,9 @@ class TooltipView extends ComponentView {
                     cbParams.axisIndex = axisItem.axisIndex;
                     cbParams.axisType = axisItem.axisType;
                     cbParams.axisId = axisItem.axisId;
-                    cbParams.axisValue = axisHelper.getAxisRawValue(
+                    cbParams.axisValue = formatUtil.encodeHTML(axisHelper.getAxisRawValue(
```

Comment:
I think it's very common that chars like '&', '"' exist in the strings. We should not do such a general encoding. At least we should use an option to enable it optionally.

Reply:
> we should use an option to enable it optionally.
You mean add an new option for user?

Reply:
Yes

Reply:
I found I thought it wrong. It won't be an issue if we escaped '&', '"' in the strings. It will be displayed normally in HTML. In this case, I think an extra option is not necessary.